### ISSUE TITLE
Always returns to 'Getting Started' after I exit brackets

### ISSUE THREAD

**Issue by LeonardoHello** [adobe/brackets#13997]
Hello.
Before, after I open a folder on '**Open Folder...**' button, the folder always stays there even after I exit brackets, and now I don't know if I accidentally did something in options but it just disappears and it returns to **'Getting Started'** and I have to open the folder again after opening brackets.
It's not a big problem I just want brackets to save every folder I open.
thanks  :100:  :100:  :100:

**Comment by petetnt** [comment]
Hi @LeonardoHello,

sounds pretty weird. Might have something to do with folder permissions. Can you confirm that you can see `brackets.json` in one of the following folders:

Mac: ~/Library/Application Support/Brackets
Win: C:\Users\<user>\AppData\Roaming\Brackets
Linux: ~/.config/brackets

**Comment by LeonardoHello** [comment]
Yes I can. I am on windows 7 btw, if that makes any difference.

**Comment by LeonardoHello** [comment]
One thing I neglected to mention is that every time I open brackets, it tells me I opened it for the first time. Because when I click on the '**Live Preview**' button it shows me a little message of how it works and even on the last line of the message it tells me "(You'll only see the message once.)", and I see that every time after I exit brackets...

**Comment by LeonardoHello** [comment]
hello,

Yes I can. I am on windows 7 btw, if that makes any difference. And one
thing I neglected to mention is that every time I open brackets, it tells
me I opened it for the first time. Because when I click on the 'Live Preview'
button it shows me a little message of how it works and even on the last
line of the message it tells me "(You'll only see the message once.)", and
I see that every time after I exit brackets...
